### DISCUSSION TITLE
Our first production Crawly spider

### DISCUSSION THREAD

**Discussion by dogweather** [elixir-crawly/crawly#231] [Show and tell]
https://github.com/public-law/oregon-revised-statutes-crawler

Code review / critique is appreciated.

Coding it was a very smooth experience. No unpleasant surprises; everything worked as expected. The [Domo](https://github.com/public-law/oregon-revised-statutes-crawler) library and the new [dbg](https://blog.appsignal.com/2022/09/13/elixir-1-14-better-debugging-with-dbg-and-more.html) debugger contributed the smooth experience. In some ways, it was easier than Scrapy/Python.

The spider creates a 60mb JSON lines file. It uses one API call and approx. 500 web pages as its source.

**Comment by serpent213** [comment]
Thank you for the example, learned a few things!

Unfortunately the service seems to be down right now...
